lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandler.java

```diff
@@ -247,52 +248,47 @@ private void sendIpvVcReceivedAuditEvent(
-                            AuditEventTypes.IPV_F2F_CRI_VC_CONSUMED,
-                            configService.getParameter(ConfigurationVariable.COMPONENT_ID),
-                            auditEventUser,
-                            null,
-                            getRestrictedAuditDataForF2F(vc));
-            auditService.sendAuditEvent(auditEvent);
-        }
+        AuditEvent auditEvent =
+                AuditEvent.createWithoutDeviceInformation(
+                        AuditEventTypes.IPV_ASYNC_CRI_VC_CONSUMED,
+                        configService.getParameter(ConfigurationVariable.COMPONENT_ID),
+                        auditEventUser,
+                        new AuditExtensionsCredentialIssuerId(cri.toString()),
+                        getRestrictedAuditDataForF2F(vc));
+        auditService.sendAuditEvent(auditEvent);
     }
 
     private void sendIpvVcErrorAuditEvent(ErrorAsyncCriResponse errorAsyncCriResponse, Cri cri) {
-        if (Cri.F2F.equals(cri)) {
-            AuditEventUser auditEventUser =
-                    new AuditEventUser(errorAsyncCriResponse.getUserId(), null, null, null);
-
-            AuditExtensionErrorParams extensionErrorParams =
-                    new AuditExtensionErrorParams.Builder()
-                            .setErrorCode(errorAsyncCriResponse.getError())
-                            .setErrorDescription(errorAsyncCriResponse.getErrorDescription())
-                            .build();
-
-            AuditEvent auditEvent =
-                    AuditEvent.createWithoutDeviceInformation(
-                            AuditEventTypes.IPV_F2F_CRI_VC_ERROR,
-                            configService.getParameter(ConfigurationVariable.COMPONENT_ID),
-                            auditEventUser,
-                            extensionErrorParams);
-            LOGGER.info(
-                    LogHelper.buildLogMessage("Sending audit event IPV_F2F_CRI_VC_ERROR message."));
-            auditService.sendAuditEvent(auditEvent);
-        }
+        AuditEventUser auditEventUser =
+                new AuditEventUser(errorAsyncCriResponse.getUserId(), null, null, null);
+
+        AuditExtensionErrorParams extensionErrorParams =
+                new AuditExtensionErrorParams.Builder()
+                        .setErrorCode(errorAsyncCriResponse.getError())
+                        .setErrorDescription(errorAsyncCriResponse.getErrorDescription())
+                        .setCredentialIssuerId(cri.toString())
+                        .build();
+
+        AuditEvent auditEvent =
+                AuditEvent.createWithoutDeviceInformation(
+                        AuditEventTypes.IPV_ASYNC_CRI_VC_ERROR,
+                        configService.getParameter(ConfigurationVariable.COMPONENT_ID),
+                        auditEventUser,
+                        extensionErrorParams);
+        LOGGER.info(
+                LogHelper.buildLogMessage("Sending audit event IPV_ASYNC_CRI_VC_ERROR message."));
```

Comment:
I know this was there before but we probably don't need this log? looks like we're already logging the error itself above